boxcli/root.go

```diff
@@ -51,17 +51,21 @@ func RootCmd() *cobra.Command {
 func Execute(ctx context.Context, args []string) int {
 	defer debug.Recover()
 	exe := midcobra.New(RootCmd())
-	exe.AddMiddleware(midcobra.Telemetry(&midcobra.TelemetryOpts{
-		AppName:      "devbox",
+	appName := "devbox"
+	exe.AddMiddleware(midcobra.Sentry(&midcobra.SentryOpts{
+		AppName:    appName,
+		AppVersion: build.Version,
+		SentryDSN:  build.SentryDSN,
+	}))
+	exe.AddMiddleware(midcobra.Segment(&midcobra.SegmentOpts{
+		AppName:      appName,
 		AppVersion:   build.Version,
-		SentryDSN:    build.SentryDSN,
 		TelemetryKey: build.TelemetryKey,
```

Comment:
SegmentKey?

Reply:
With the middleware being Segment, I agree this should become `SegmentKey`. Make sure to update CICD so that `build.SegmentKey` works.